src/app/pages/app-list/app-list.component.scss

```
@@ -1111,6 +1115,53 @@
+      right: 10px;
+      left: auto;
+    }
+  }
+
+  .badge {
+    padding: 4px 8px;
+    border-radius: 4px;
+    font-size: 0.75rem;
+    font-weight: bold;
+
+    line-height: 1;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)